python/pyspark/sql/connect/plan.py

```diff
@@ -623,6 +623,30 @@ def plan(self, session: "SparkConnectClient") -> proto.Relation:
         return plan
 
 
+class DeduplicateWithinWatermark(LogicalPlan):
```

Comment:
We don't ned this anymore, right? we can add the flag to `Deduplicate` above to match the rest of the PR.

Reply:
yeah I noticed that as well, doing the change now

Reply:
updated